sdk/src/main/java/com/qonversion/android/sdk/internal/userProperties/PendingUserPropertiesStorage.kt

```diff
@@ -0,0 +1,12 @@
+package com.qonversion.android.sdk.internal.userProperties
+
+internal interface PendingUserPropertiesStorage {
+
+    fun getProperties(): Map<String, String>
+
+    fun save(key: String, value: String)
+
+    fun delete(key: String)
```

Comment:
I think it would be good to have a `delete` method for an array of keys. The single key method is also needed for interface I think but won't be used in the controller.